lib/loader/loader.js

```diff
@@ -110,12 +112,45 @@ var loader, define, requireModule, require, requirejs;
 
   }
 
+  if (DEBUG) {
+    var deprecationLogger = function () {
+      console.warn.apply(console, arguments);
+    };
+
+    var constructMakeDefaultExportDeprecation = function (id) {
+      return 'The `' + id + '` module does not define a default export, but loader.js ' +
+        'generated one anyway. This behavior is deprecated and will be removed in v5.0.0.';
+    };
+
+    Object.defineProperty(requirejs, 'deprecationLogger', {
```

Comment:
Why do we need this? If it is just for the testing, can we tweak it to just mock console?